src/FlattenException.php

```diff
@@ -74,120 +74,88 @@ public function __construct(\Exception $exception)
     }
 
     /**
```

Comment:
What happened in this file?

Reply:
It's code rearrangment. The first thing I always do working on another "yii"-project is loading [yii2-coding-standards](https://github.com/yiisoft/yii2-coding-standards) and let IDE reformat code.